ext/standard/tests/strings/chr_error.phpt

```diff
@@ -1,34 +0,0 @@
---TEST--
```

Comment:
Removed as per https://externals.io/message/103886.

Reply:
In this instance the tests should stay, because chr() has non-standard behavior (it returns a null byte).